library/init/core.lean

```diff
@@ -168,10 +178,18 @@ quot.lift f _ (quot.mk a) ~~> f a
 -/
 init_quotient
 
-/-- Heterogeneous equality.
+/--
+Heterogeneous equality.
+
 It's purpose is to write down equalities between terms whose types are not definitionally equal.
```

Comment:
```suggestion
Its purpose is to write down equalities between terms whose types are not definitionally equal.
```

Reply:
(phew, I thought at first I introduced this, but it's been there all along!)